reposerver/src/test/scala/com/advancedtelematic/tuf/reposerver/client/ReposerverHttpClientSpec.scala

```diff
@@ -91,3 +91,11 @@ class ReposerverHttpClientSpec extends TufReposerverSpec
                      Sha256Digest.digest("hi".getBytes), 42, BINARY).failed.futureValue shouldBe ReposerverClient.NotFound
   }
 }
+
+class RsaReposerverHttpClientSpec extends ReposerverHttpClientSpec {
```

Comment:
I've added a commit with a simpler FakeKeyserverClient (Rsa/EdFakeKeyserverClient can disappear completely) and changed `OfflineSignedRoleStorageSpec` as a example to also make it simpler. Maybe you can give it a quick review before I change the other tests.

Reply:
I don't know why I need castToKeyPair instead of toKeyPair here: https://github.com/advancedtelematic/ota-tuf/pull/156/commits/9147259d129740eb374a00df5446c254368727a1#diff-16fdb54f74b5f6fd9baddd61e899fbfaR183

Reply:
Never mind, toKeyPair works now with my latest changes.